colibre/auto_plotter/stellar_mass_halo_mass.yml

```diff
@@ -588,3 +588,152 @@ stellar_mass_halo_mass_M200_centrals_30:
     - filename: GalaxyStellarMassHaloMass/Moster2013.hdf5
     - filename: GalaxyStellarMassHaloMass/Schaye2015_Ref_100_M200.hdf5
     - filename: GalaxyStellarMassHaloMass/Read2017.hdf5
+
+stellar_mass_halo_mass_M200_centrals_projected_10:
+  type: "scatter"
+  select_structure_type: 10
+  comment: "Centrals only"
+  legend_loc: "upper left"
+  x:
+    quantity: "masses.mass_200crit"
+    units: Solar_Mass
+    start: 1e13
+    end: 5e15
+  y:
+    quantity: "derived_quantities.projected_stellar_mass_10_kpc"
+    units: Solar_Mass
+    start: 5e10
+    end: 1e13
+  median:
+    plot: true
+    log: true
+    adaptive: true
+    number_of_bins: 10
+    start:
+      value: 1e13
+      units: Solar_Mass
+    end:
+      value: 5e15
+      units: Solar_Mass
+  metadata:
+    title: Stellar Mass-Halo Mass relation (10 kpc 2D aperture, $M_{200,crit}$, centrals only)
+    caption: Includes only central haloes, and excludes the contribution of satellites in projection, for both the simulated and observed data. The halo masses in observations by DeMaio et al. (2020) and the semi-empirical model by Huang et al. (2021) were converted from the $M_{500}$ and $M_\mathrm{BN98}$ definitions, respectively, to a $M_{200}$ definition (critical overdensity), by assuming an NFW profile and concentration $c=5$. The halo masses from Golden-Marx et al. (2022) were converted from a mean overdensity to critical overdensity (factor 200) definition. None of the halo masses are corrected for (hydrostatic and other) bias. The IMF from observations by Golden-Marx et al. (2022) was converted from Salpeter to Chabrier. Cosmology corrections were not needed. The data that are of intermediate-redshift objects can be used for comparison here because the evolution of BCG stellar cores is observed to be negligible in this regime. The Huang et al. (2021) model is tailored to groups and clusters, and uses aperture definitions for stellar masses, not only for its predictions but also for the observational data used for calibration.
```

Comment:
In the caption, shouldn't it be Huang et al. (2022) instead of Huang et al. (2021)?

The same comment applies to captions of the other figures too.

Reply:
Yes, I had been referring to this paper as being published in 2021 when I was starting to add these changes, but then when I found the full reference, the publication date was actually 2022. Clearly I didn't correctly update it everywhere.